models/models.ts

```diff
@@ -87,7 +87,7 @@ export interface CoapRequestParams {
     port?: number
     method?: CoapMethod
     confirmable?: boolean
-    observe?: 0 | 1 | boolean
```

Comment:
I was just passing by but why do you want to disallow people to use `false`? sorry for bothering I'm just curious. 😃

Reply:
Good point actually ;) I think the rationale behind it was that a boolean value of `false` does not have any effect at the moment. It could be mapped to a value of 1 (i. e. unobserve) but that might constitute a breaking change.

In any case, I changed the type back to `boolean`, realizing that it might otherwise introduce other incompatabilities with node-wot.